### ISSUE TITLE
Suggestion: Avoid crashing when trying to delete a non-exisiting directory.

### ISSUE THREAD

**Issue by Raizanad** [RimSort/RimSort#445]
Adding to #438 
RimSort also crashes if you try to delete a directory that was already moved/delete manually in file explorer.
Ideally the App shouldn't crash, just ignore the folder/command, and optionally based on user settings refresh.

**Comment by hk21702** [comment]
Should have been fixed with #437
